src/plugins/saved_objects_management/public/management_section/saved_objects_table_page.tsx

```diff
@@ -106,7 +106,11 @@ const SavedObjectsTablePage = ({
         }}
         canGoInApp={(savedObject) => {
           const { inAppUrl } = savedObject.meta;
-          return inAppUrl ? Boolean(get(capabilities, inAppUrl.uiCapabilitiesPath)) : false;
+          return inAppUrl
+            ? !inAppUrl.uiCapabilitiesPath
```

Comment:
Can we please extract this nested ternary into something that's more readable (and therefore easier to reason about)?

Reply:
Done.